ASReviewLAB.qmd

```diff
@@ -185,17 +182,17 @@ The following five papers are known to be relevant:
 -   Trajectories of trauma symptoms and resilience in deployed US
     military service members: Prospective cohort study (DOI: [10.1192/bjp.bp.111.096552](https://doi.org/10.1192/bjp.bp.111.096552){target="_blank"})
 
-To add the relevant records, you click on ‘Search’, copy and paste the titles
+To add the relevant records, you click on ‘`Search`’, copy and paste the titles
 of these relevant records one by one in the search bar and add them as
 relevant.
 
-![](https://raw.githubusercontent.com/asreview/asreview/main/docs/images/setup_prior_knowledge_random_validate.png){width=95%}
+![](images/ASReviewLAB/step_5.png){width=95%}
```

Comment:
I'm okay with this revert but not taking care of it. I won't update the screenshots or inform you when the UI changes (and changes are expected more often in the future).

Reply:
I would prefer the new solution with taking the images from the asreview repo, but I am missing images. 

Full page with no selections. I want to point the user towards the create button: 
![](https://jteijema.github.io/asreview-academy/images/ASReviewLAB/step_3.png)

Create a new project without the mode selector dropdown open (it obscures the other textboxes):
![](https://jteijema.github.io/asreview-academy/images/ASReviewLAB/step_3b.png)

An updated dataset selector. The current image does not show the latest version. Or if it does, my order of datasets is different (synergy first, then nature datasets).
![](https://jteijema.github.io/asreview-academy/images/ASReviewLAB/step_4.png)

The model selection screen:
![](https://jteijema.github.io/asreview-academy/images/ASReviewLAB/step_6.png)

If these images were available, then I would absolutely support taking the images from the asreview repo, but having images from two different places will end up in differences in resolution and device, which might confuse readers.

Reply:
Isn't it possible to add the required images to https://github.com/asreview/asreview/tree/main/docs/images?